denops/gin/command/edit/write.ts

```diff
@@ -65,8 +65,11 @@ export async function exec(
     restore = () => Deno.remove(original);
   }
   try {
+    const stat = await Deno.stat(f);
     await fs.ensureFile(original);
-    await Deno.writeTextFile(original, `${content.join("\n")}\n`);
+    await Deno.writeTextFile(original, `${content.join("\n")}\n`, {
+      mode: stat.mode ?? undefined,
```

Comment:
We don't need `?? undefined`

Reply:
stat.mode contains `null`. It intention to convert null to undefined.
I noticed hard to understand in this review. I'll rewrite it. sorry.